CppCoreGuidelines.md

```diff
@@ -9224,6 +9224,9 @@ Specifying the underlying type is necessary in forward declarations of enumerati
 
     enum Flags : char { /* ... */ };
 
+or to ensure that values of type that can have a specified bit-precision:
```

Comment:
Would "or to ensure that values of that type have a specified bit-precision:" make more sense?